src/tools/annotation/TextMarkerTool.js

```diff
@@ -117,10 +117,6 @@ export default class TextMarkerTool extends BaseAnnotationTool {
     return distanceToPoint < 10 || insideBoundingBox;
   }
 
-  updateCachedStats(image, element, data) {
```

Comment:
Again, need to double check we're not calling this generically; but I think we should be safe